web/exercises/replyGraphGelly.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: Gelly API - Reply Graph with Page Rank
+permalink: /exercises/replyGraphGelly.html
+---
+The task of the “Reply Graph with Page Rank” exercise is to define a graph using Gelly-API and then to analyze the structure of the Flink community by running a page Rank algorithm over it.
+The input data set for graph generation is the output of the  [Reply Graph]({{ site.baseurl }}/exercises/replyGraph.html)exercise.
+
+
+### Input Data
+
+This exercise uses the output of the [Reply Graph]({{ site.baseurl }}/exercises/replyGraph.html) exercise as input (`CsvInputFormat`).
+
+The input data can be read as `DataSet<Tuple3<String, String, Integer>>`. The first field is the sender email address of the reply mail, the second field is the sender email address of the mail that was replied to, and the third field is the number of reply connections between these two email addresses. When printed, the data set should look like this:
+
+~~~
+(fhueske@gmail.com,hongbosoftware@163.com,1)
+(fhueske@gmail.com,kostas.tzoumas@tu-berlin.de,9)
+(fhueske@gmail.com,mneumann@spotify.com,5)
+(fhueske@gmail.com,sewen@apache.org,42)
+~~~
+
+
+### Expected Output
+
+
+The result of the exercise should be a `DataSet<Vertex<String,Double>>`. The first field indicates the vertex id i.e. email address, the second field is its associated rank.
+When printed, the data set should look like this:
+
+~~~
+gyula.fora@gmail.com	0.023443367885291905
+
+cos@apache.org	0.001387967316018071
+~~~
+
+### Implementation Hints
+
+#### Program Structure
+
+This exercise can be solved in three steps.
+
+1. Read each tuple `DataSet<Tuple3<String, String, Double>>` of the replyGraph exercise as an Edge.
```

Comment:
You list three steps here but explain only two steps later ("Generating a graph with weighted edges", "Calculating page rank").